x-pack/plugins/observability_ai_assistant/public/functions/visualize_esql.tsx

```diff
@@ -51,10 +51,22 @@ enum ChartType {
   Waffle = 'Waffle',
 }
 
-interface VisualizeLensResponse {
+interface VisualizeQueryResponsev0 {
```

Comment:
What does the `v0` and `v1` signify?

Reply:
version 0 and version 1 - because data is persisted to the conversation, the render function needs to deal with older versions as well